charts/kyverno/templates/reports-controller/deployment.yaml

```diff
@@ -1,3 +1,15 @@
+{{- if and (eq .Values.reportsController.enabled true) (eq .Values.crds.groups.wgpolicyk8s.clusterpolicyreports false) }}
+{{- fail "CRD clusterpolicyreports disabled while reportsController enabled" }}
+{{- end }}
+{{- if and (eq .Values.reportsController.enabled true) (eq .Values.crds.groups.wgpolicyk8s.clusterpolicyreports false) }}
+{{- fail "CRD clusterpolicyreports disabled while reportsController enabled" }}
+{{- end }}
+{{- if and (eq .Values.reportsController.enabled true) (eq .Values.crds.groups.kyverno.backgroundscanreports false) }}
+{{- fail "CRD backgroundscanreports disabled while reportsController enabled" }}
+{{- end }}
+{{- if and (eq .Values.reportsController.enabled true) (eq .Values.crds.groups.kyverno.clusterbackgroundscanreports false) }}
+{{- fail "CRD backgroundscanreports disabled while reportsController enabled" }}
```

Comment:
```suggestion
{{- fail "CRD clusterbackgroundscanreports disabled while reportsController enabled" }}
```

Reply:
updaterequest but when decoupled, these two will be related. Currently as admissioncontroller needs updaterequest as well we will install it by default and have no check. @realshuting